pkg/utils/grpcutil/grpcutil_test.go

```diff
@@ -1,14 +1,22 @@
 package grpcutil
 
 import (
+	"fmt"
 	"os"
+	"os/exec"
+	"path"
 	"testing"
 
 	"github.com/pingcap/errors"
 	"github.com/stretchr/testify/require"
 	"github.com/tikv/pd/pkg/errs"
 )
 
+var (
+	certPath   = "../../../tests/integrations/client/cert"
```

Comment:
It's better to use an absolute path. the script generates can to a `tmp` path.

Reply:
> It's better to use an absolute path

It may be necessary to merge the master and then change it to absolute path
> the script generates can to a tmp path.

yes, fixed